lib/livebook_web/live/session_live/shortcuts_component.ex

```diff
@@ -25,7 +25,8 @@ defmodule LivebookWeb.SessionLive.ShortcutsComponent do
       %{seq: ["ej"], desc: "Evaluate cells below"},
       %{seq: ["ex"], desc: "Cancel cell evaluation"},
       %{seq: ["ss"], desc: "Toggle sections panel"},
-      %{seq: ["sn"], desc: "Show notebook settings"}
+      %{seq: ["sn"], desc: "Show notebook settings"},
```

Comment:
If we want to show both, then maybe this one should say "Show notebook file settings" then?

Reply:
I think having both shortcuts would be a good option. If we only have 1 shortcut, we have to use a mouse/ trackpad to reach the Runtime tab (after opening the notebook setting modal). 😀

Reply:
Should I also change the shortcut from s+n to s+f if @jonatanklosko is ok to have both shortcuts?

Reply:
@josevalim as we start adding more tabs, I don't think all are gonna be important enough to deserve a shortcut (i.e. I am more than fine having shortcuts for everything, but it's not necessarily the best approach given we list all the shortcuts ^^). Runtime settings are definitely important, especially when you need to reconnect. I'd stick to `sr` for now, WDYT?

Reply:
Sounds good to me! We also never know the future, it may be that we end-up splitting those too into separate modals altogether. I keep thinking the button on the lower right should be the one used for opening the file settings: i.e. it will always be clickable. And if you want to "Save now", we can make the "Save" button clickable if a file is selected and there are pending saves. But this is something we can revisit later!

Reply:
And we can also hijack the Ctrl+S shortcut for saving.